plugin-server/tests/worker/ingestion/session-recording.test.ts

```diff
@@ -0,0 +1,113 @@
+import { StatsD } from 'hot-shots'
+import { DateTime } from 'luxon'
+
+import { connectObjectStorage, ObjectStorage } from '../../../src/main/services/objectStorage'
+import { TimestampFormat } from '../../../src/types'
+import { castTimestampOrNow } from '../../../src/utils/utils'
+import { processSnapshotData } from '../../../src/worker/ingestion/session-recordings'
+
+describe('session recordings', () => {
```

Comment:
:+1: for this test file. However we're testing a specific function here:
```suggestion
describe('processSnapshotData()', () => {
```